Jenkinsfile

```diff
@@ -102,10 +102,11 @@ pipeline {
   post {
     failure {
       script {
-        if (!isPr) {
+        boolean buildingMain = env.JOB_NAME == "${service}/main"
+        if (buildingMain || params.RELEASE_BUILD) {
```

Comment:
We want notifications for main and release builds specifically.  We don't want notifications for PRs or full builds on a PR branch.